### ISSUE TITLE
Reolink TrackMix PoE - second channel

### ISSUE THREAD

**Issue by phil-schneider** [fwestenberg/reolink_dev#597]
**Describe the bug**
Hello,
I have a Reolink TrackMix PoE camera with dual lens. 
https://reolink.com/ch/product/reolink-trackmix-poe/
The camera has two channel / streams / videos inside the android app and also on the mac app.

But somehow I can only see one camera inside the plugin.
I already tried to reconfigure the integration, but there is no option on switching channel. Switching stream does not change anything.

What am I missing?

**Expected behavior**
Expectation on an opensource plugin ...
Dreaming... Each channel is a separate camera, so in the case of this camera model, there will be two cameras.

**Screenshots**
<img width="246" alt="image" src="https://user-images.githubusercontent.com/2550609/202764787-ee843c92-be15-4330-a0ca-200f1ec8ef3e.png">

![image](https://user-images.githubusercontent.com/2550609/202766354-adab4d61-338b-463e-bc8d-e9f52a6bb423.png)

**Environment:**
Please provide useful information about your environment, like:
 - Home Assistant 2022.11.3
 - Model Reolink TrackMix PoE
 - Hardware No. IPC_529SD78MP
- Config Version v3.0.0.0
- Firmware Versionv3.0.0.1123_22071300

**Additional context**
Thanks for your help

**Comment by phil-schneider** [comment]
I even found the url for the direkt link to the second stream, maybe that helps

rtsp://MYUSER:MYSECRET@192.168.xxx.xxx:554/h265Preview_02_main

**Comment by phil-schneider** [comment]
Hello @cvjensen,
Yes the sensors work for me. But I can not say how good they are. Have not tested much more and camera is currently not active.

**Comment by dhavaldesai-HA** [comment]
Similar issue on Reolink Duo Camera. Would be great if someone can help.

**Comment by cvjensen** [comment]
> Similar issue on Reolink Duo Camera. Would be great if someone can help.

I changed to the reolink_cctv plugin but noticed that my internal adress wasnt corrent nor did I use http internal . Which is required

**Comment by MiAutomations** [comment]
@dhavaldesai-HA and @phil-schneider 

You can find more information about the Dual lens integration in Home Assistant in the following link (The site is in Portugues) You can Translate with google Translate  

https://sites.google.com/view/mi-automations/menu-inicial/cameras-ip/reolink-dual-lenses-home-assistant

**Comment by Homergy** [comment]
> Hey @phil-schneider sorry for commenting off topic .. I just saw that you are the only one with the same trackmix as I just got. Are your smart sensors relieable?
> 
> ![image](https://user-images.githubusercontent.com/9024493/204215060-c97c3648-b52c-48ae-9eb6-ceb9d7f7b672.png)

I have the exact same issue. connecting to the Trackmix via Reolink NVR.

**Comment by MiAutomations** [comment]
> > Hey @phil-schneider sorry for commenting off topic .. I just saw that you are the only one with the same trackmix as I just got. Are your smart sensors relieable?
> > ![image](https://user-images.githubusercontent.com/9024493/204215060-c97c3648-b52c-48ae-9eb6-ceb9d7f7b672.png)
> 
> I have the exact same issue. connecting to the Trackmix via Reolink NVR.

Yes they are more or less, you can also adjust the sensors on the webpage of the Camera  or over the windows app or in the app

![IMG_20221222_203841](https://user-images.githubusercontent.com/47602733/209222471-66d74fc0-0842-40c0-ac7e-5722f8dd3c52.jpg)


Cheers

**Comment by phil-schneider** [comment]
Sorry. Currently not active.
One month ago, they were active.

**Comment by rw377** [comment]
Adding on here, I also have a Trackmix PoE. The sensors are continuously becoming unavailable and then available again similar to https://github.com/fwestenberg/reolink_dev/issues/445 which is about a year old with no solution.

**Comment by cpainchaud** [comment]
solution is to ask Reolink to fix their camera crashing, that's what is causing the unavailable/available flashing .... happens on mine about a hundred times a day

**Comment by rw377** [comment]
@cpainchaud so it is the camera itself crashing? I'm not seeing streams via Blue Iris lose signal, its just the Person/Pet/Vehicle sensors become unavailable on the integration. If the hardware itself was restarting wouldn't signal be lost or is it something in the camera software causing an issue?

**Comment by cpainchaud** [comment]
API server itself can crash and reboot while video stream remains fine. I see that on mine on all day long.
